.github/workflows/main.yaml

```diff
@@ -129,6 +146,39 @@ jobs:
           body_path: ./release-artifacts/changelog.md
           draft: ${{ startsWith(steps.get_tag.outputs.git_tag, 'nightly') != true }}
           prerelease: ${{ startsWith(steps.get_tag.outputs.git_tag, 'nightly') }}
+  push-image:
+    name: Push Image
+    needs:
+      [create-release]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: |
+            ${{ IMAGE_NAME }}:${{ github.sha }},
```

Comment:
This still has a comma at the end (I saw it was removed, and then made its way back...). I think it's not correct, as mentioned previously, but I'm not 100% sure. If it's intentional let me know.

Reply:
OK, I see this was fixed in the last commit.